app/routes/rental_routes.py

```diff
@@ -0,0 +1,51 @@
+from app import db
+from app.models.rental import Rental
+from app.models.customer import Customer
+from app.models.video import Video
+from flask import Blueprint, request, jsonify
+from datetime import datetime, timedelta
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def check_out_video():
+    request_body = request.get_json()
+
+    try:
+        customer = Customer.query.get_or_404(request_body["customer_id"])
+        video = Video.query.get_or_404(request_body["video_id"])
+    except KeyError:
+        return jsonify(None), 400
```

Comment:
since you are using `get_or_404` here, technically you wouldn't need to use the `try` as well. `get_or_404` will take care of any errors for you and return them in an HTML response body.

I think sticking with `try`/`except` and a basic `get` would be a better combo!